react-app/src/components/CommunityStatus/CommunityStatusRegular.tsx

```diff
@@ -62,9 +62,10 @@ export const CommunityStatusRegular: React.FC<CommunityStatusRegularProps> = (
         "flex",
         "flex-col",
         "gap-y-3",
-        "sm:gap-y-0",
+        "sm:gap-y-3",
```

Comment:
would this create extra spacing vertically on desktop view?

also there's `"gap-y-3"` already so no need to put it again

Reply:
> would this create extra spacing vertically on desktop view?

seems no

<img width="875" alt="image" src="https://user-images.githubusercontent.com/106721852/178964082-21e714e0-6a88-4d22-affc-7b858dfba926.png">